SO_properties.py

```diff
@@ -2522,9 +2523,9 @@ class SOProperties(HaloProperty):
             "starOfrac",
             "starFefrac",
             "gasmetalfrac_SF",
-            "concentration",
+            "concentration_unsoft",
             "concentration_soft",
```

Comment:
Does this not need changing from `concentration_soft` to `concentration`?

Reply:
I'm keeping everything explicit, so the softened concentration function is still called concentration_soft